blocks_vertical/event.js

```diff
@@ -27,6 +27,48 @@ goog.require('Blockly.Colours');
 goog.require('Blockly.constants');
 goog.require('Blockly.ScratchBlocks.VerticalExtensions');
 
+Blockly.Blocks['event_whentouchingobject'] = {
+  /**
+   * Block to when a sprite is touching an object.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit({
+      "message0": "when this sprite touches %1",
```

Comment:
This string should go in msg/messages.js so that this block can get translated. (E.g. 'Blockly.Msg.EVENT_WHENFLAGCLICKED').

I realize we might have this same problem in other places, so we might need to do an audit of that...